project-canvas/src/translations/de.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Common": {
+    "btn-goBack": "Zurück",
+    "yes": "Ja",
+    "no": "Nein"
+  },
+
+  "Login": {
+    "choseProvider": "Bitte wählen Sie einen Anbieter"
```

Comment:
The name "Login" is in capital letter because it is the Name of the component. The name of the element like a button has the prefix "btn" to indicate that it is a button. It is easyer to see if the text belongs to a button. If you dont like it i can alos do somthing like:
`"Login": {
   "button": {
      "goBack" : " Go Back"`
So we can access the String using Login.button.goBack

Reply:
That would be better and more consistent, thanks 👍